app/Resources/FOSUserBundle/translations/validators.en.yml

```diff
@@ -0,0 +1,26 @@
+fos_user:
```

Comment:
Those are intended for the validation message in FOSUserBundle. We can have in translation but it is much ideal to be there. And also that is how you override the translation of FOSUserBundle if you don't know it.